docs/howtos/testing-a-rust-component.md

```diff
@@ -109,25 +94,25 @@ Each component that implements a sync engine should have a corresponding suite o
 
 ### Android Components Test Suite
 
-It's important that changes in application-services are tested against upstream consumer code in the
+It's important that changes in `application-services` are tested against upstream consumer code in the
 [android-components](https://github.com/mozilla-mobile/android-components/) repo. This is currently
 a manual process involving:
 
 * Configuring your local checkout of android-components to [use your local application-services
-  build](./working-with-reference-browser.md).
+  build](./locally-published-components-in-fenix.md).
 * Running the android-components test suite via `./gradle test`.
 * Manually building and running the android-components sample apps to verify that they're still working.
 
 Ideally some or all of this would be automated and run in CI, but we have not yet invested in such automation.
 
 ## Test Coverage
 
-Lamentably, we do not measure or report on code test coverage.
+Lamentably, we have limited reporting on code coverage
```

Comment:
I think  we can change this - our current coverage is OK and the linked issue is closed. I think the biggest limitation is that we can't get coverage caused by other components - eg, we don't know how must extra coverage is caused by, say, a-c tests. So hand-waving about that might make sense.